### PR TITLE
KomBurSROCK2

### PR DESCRIPTION
**Pull request by deeepeshthakur**
@ChrisRackauckas 
Implements `KomBurSROCK2` from [Weak Second Order S-ROCK Methods for Stratonovich Stochastic Differential Equations](https://doi.org/10.1016/j.cam.2012.01.033)
#73 
Note: I have to verify the scond order and add tests.

### CODE REVIEW

**Comment by deeepeshthakur** [comment]
I'm getting plots like this. 
```
using Plots
using StochasticDiffEq, DiffEqProblemLibrary, Test, Random, DiffEqDevTools
using DiffEqProblemLibrary.SDEProblemLibrary: importsdeproblems; importsdeproblems()
import DiffEqProblemLibrary.SDEProblemLibrary: prob_sde_linear_stratonovich, prob_sde_2Dlinear_stratonovich
Random.seed!(100)
dts = (1/2) .^ (10:-1:2)

prob = prob_sde_linear_stratonovich
sim  = test_convergence(dts,prob,KomBurSROCK2(),numMonte=Int(2e3))
plot(sim,dpi=200,linewidth=1,legend=:bottomright,legendfontsize=4)
```
image

**Comment by deeepeshthakur** [comment]
I'm getting order 2.0 now
```
using StochasticDiffEq, DiffEqProblemLibrary, Test, Random, DiffEqDevTools, Plots
using DiffEqProblemLibrary.SDEProblemLibrary: importsdeproblems; importsdeproblems()
import DiffEqProblemLibrary.SDEProblemLibrary: prob_sde_linear_stratonovich, prob_sde_2Dlinear_stratonovich
Random.seed!(100)
dts = 1 ./2 .^(10:-1:2) #14->7 good plot

prob = prob_sde_linear_stratonovich

sim = test_convergence(dts,prob,KomBurSROCK2(),numMonte=Int(1e2),weak_timeseries_errors=true)
plot(sim,dpi=200,linewidth=1,legend=:bottomright,legendfontsize=4)
```
image


```
sim = test_convergence(dts,prob,KomBurSROCK2(),numMonte=Int(1e3),weak_timeseries_errors=true)
plot(sim,dpi=200,linewidth=1,legend=:bottomright,legendfontsize=4)
```
image


```
prob = prob_sde_2Dlinear_stratonovich

sim = test_convergence(dts,prob,KomBurSROCK2(),numMonte=Int(1e2),weak_timeseries_errors=true)
plot(sim,dpi=200,linewidth=1,legend=:bottomright,legendfontsize=4)
```
image


```
sim = test_convergence(dts,prob,KomBurSROCK2(),numMonte=Int(1e3),weak_timeseries_errors=true)
plot(sim,dpi=200,linewidth=1,legend=:bottomright,legendfontsize=4)
```
image

File: test/stratonovich_convergence_tests.jl [hunk 1]
```diff
@@ -32,6 +32,9 @@ sim  = test_convergence(dts,prob,ImplicitRKMil(interpretation=:Stratonovich),num
 sim  = test_convergence(dts,prob,SROCK1(interpretation=:Stratonovich),numMonte=Int(2e2))
 @test abs(sim.𝒪est[:l2]-1) < 0.15
 
+sim  = test_convergence(dts,prob,KomBurSROCK2(),numMonte=Int(2e2))
+@test abs(sim.𝒪est[:final]-2) < 0.20
```
Comment (ChrisRackauckas) [inline test/stratonovich_convergence_tests.jl]: this is strong convergence?

Comment (deeepeshthakur) [reply test/stratonovich_convergence_tests.jl]: This is a linear problem that’s why the approx 2.0 order convergence.

**Comment by ChrisRackauckas** [comment]
We need to update AppVeyor so it runs a more useful subset again.

**Pull request merged**